src/TargetingSystem.cpp

```diff
@@ -0,0 +1,69 @@
+#include "TargetingSystem.h"
+
+
+TargetingSystem::TargetingSystem(int word_length_lb, int word_length_ub, std::string color_hex)
+    : word_length_lb_(word_length_lb)
+    , word_length_ub_(word_length_ub)
+    , color_hex_(color_hex) {
+    setup_local_dict("placeholder");
```

Comment:
Can you pass in "assets/dictionary.txt", less to change later and actually use `relative_path` arg in the func.